changelog.md

```diff
@@ -1,3 +1,88 @@
+ <a name="4.3.0"></a>
+# 4.3.0 (2019-04-08)
+[Full Changelog](https://github.com/googlechrome/lighthouse/compare/4.2.0...4.3.0)
+
+## New contributors!
+
+Thanks to @rdlabo and @yashrajbharti for their first contributions!
+
+## Clients
```

Comment:
i like the changelog groupings to be sorted by importance to the user.. something like: Report, Core, Docs, Clients, Tests, Misc, Deps

Reply:
done

Reply:
we should fix the script to make this automatic :)